api/openapi-spec/swagger.json

```diff
@@ -80582,7 +80582,7 @@
       "type": "boolean"
      },
      "volumeDevices": {
-      "description": "volumeDevices is the list of block devices to be used by the container. This is an alpha feature and may change in the future.",
+      "description": "volumeDevices is the list of block devices to be used by the container. This is a beta feature.",
```

Comment:
do we even need to say "this is a alpha/beta" feature. when we go GA we are going to have to regenerate this all over again, I don't think it is necessary to say at what stage a feature is in the API description.

Reply:
I checked through swagger.json and found out that there are many other descriptions that describe their stage of the feature. So, I think that it would be better keep this as is for this PR and may remove all of these description in the separate PR after we agreed to remove all of them.